src/core/qfieldappauthrequesthandler.cpp

```diff
@@ -83,7 +85,15 @@ bool QFieldAppAuthRequestHandler::handleLayerLogins()
       }
       else
       {
-        emit reloadEverything();
+        const QList<QgsMapLayer *> mapLayers = QgsProject::instance()->mapLayers().values();
+        for ( QgsMapLayer *mapLayer : mapLayers )
+        {
+          if ( !mapLayer->isValid() && mapLayer->dataProvider() )
+          {
+            mapLayer->setDataSource( mapLayer->source(), mapLayer->name(), mapLayer->dataProvider()->name() );
```

Comment:
If I'm not mistaken, that forces a reload of the datasource?
If that's the goal of this line, a comment would be justified, that's not immediately clear for a reader.